storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -477,7 +477,7 @@ bool backup_file_print(const char *filename, const char *message, int len) {
   MY_STAT stat;
 
   memset(&stat, 0, sizeof(stat));
-  stat.st_mtime = my_time(0);
+  stat.st_mtime = time(nullptr);
```

Comment:
my_time() vs time(nullptr) ?

Reply:
yes, this came from https://github.com/mysql/mysql-server/commit/7230b2e10e89e8df920f83fc11f12b97937b3f26
> Change all calls my_time(0) to ::time(nullptr).